examples/mxnet_imagenet_resnet50.py

```diff
@@ -136,6 +136,9 @@
 else:
     raise ValueError('Invalid lr mode')
 
+
```

Comment:
Less whitespace?

Reply:
+1 for less empty lines

Reply:
removed. only 2 blank lines now to satisfy pep8